aten/src/ATen/native/nested/NestedTensorBinaryOps.cpp

```diff
@@ -124,9 +128,8 @@ Tensor NestedTensor_elementwise_Tensor(
     }
 
     if (is_broadcastable_3d) {
-      if (!nested_tensor_impl_is_contiguous(self_ptr)) {
-        self_ptr = get_nested_tensor_impl(self.contiguous());
-      }
+      self_contiguous = self.contiguous();
```

Comment:
I think this was a potential use after free. `self.contiguous()` is a temporary and goes out of scope. This at least was the case when I was calling later on, not sure if something else was bumping ref count

Reply:
Yep, you're right. It's a pointer to a temporary object that goes out of scope as `get_nested_tensor_impl` returns.